app/assets/javascripts/redux/modules/models/responses.js

```diff
@@ -40,4 +69,39 @@ export const reducer = handleActions({
     loading: false,
     loaded: false,
   }),
+
+  [LISTEN_SIGNALS_PUT_REQUEST]: (state, action) => ({
```

Comment:
Hmm strange, i took this out and commited the final one last night but why isn't this automatically closed out?